src/index.js

```diff
@@ -49,73 +50,117 @@ function createCommand(queryName, implementationName) {
+
+      const win = cy.state('window')
+
       const consoleProps = {
         // TODO: Would be good to completely separate out the types of input into their own properties
         input: inputArr,
+        Selector: getSelector(),
+        'Applied To': getContainer(
+          options.container || prevSubject || win.document,
+        )
       }
 
-      Cypress.log({
-        $el: inputArr,
-        name: queryName,
-        message: inputArr,
-        consoleProps: () => consoleProps,
-      })
+      if (options.log) {
+        options._log = Cypress.log({
+          name: queryName,
+          message: inputArr,
+          consoleProps: () => consoleProps,
+        })
+      }
 
-      return cy
-        .window({log: false})
-        .then({timeout: waitOptions.timeout + 100}, thenArgs => {
-          const getValue = () => {
-            const value = commandImpl(thenArgs.document)
-            const result = Cypress.$(value)
-
-            // Overriding the selector of the jquery object because it's displayed in the long message of .should('exist') failure message
-            // Hopefully it makes it clearer, because I find the normal response of "Expected to find element '', but never found it" confusing
-            result.selector = `${queryName}(${queryArgument(args)})`
-
-            if (result.length > 0) {
-              consoleProps.yielded = result.toArray()
-            }
-
-            return result
-          }
-
-          const resolveValue = () => {
-            // retry calling "getValue" until following assertions pass or this command times out
-            return Cypress.Promise.try(getValue).then(value => {
-              return cy.verifyUpcomingAssertions(value, waitOptions, {
-                onRetry: resolveValue,
-              })
-            })
-          }
+      const getValue = () => {
+        const value = commandImpl(win.document)
+
+        const result = Cypress.$(value)
+        if (value && options._log) {
+          options._log.set('$el', result)
+        }
 
-          if (queryRegex.test(queryName)) {
-            // For get* queries, do not retry
-            return getValue()
-          }
+        // Overriding the selector of the jquery object because it's displayed in the long message of .should('exist') failure message
+        // Hopefully it makes it clearer, because I find the normal response of "Expected to find element '', but never found it" confusing
+        result.selector = getSelector()
 
-          return resolveValue().then(subject => {
-            // Remove the error that occurred because it is irrelevant now
-            if (consoleProps.error) {
-              delete consoleProps.error
-            }
+        consoleProps.elements = result.length
+        if (result.length === 1) {
+          consoleProps.yielded = result.toArray()[0];
+        } else if (result.length > 0) {
+          consoleProps.yielded = result.toArray();
+        }
```

Comment:
This is how other Cypress commands show `Yielded`

https://github.com/cypress-io/cypress/blob/13ebb9779d21238cae4da4b63cf6230da4a5341d/packages/driver/src/cy/commands/traversals.coffee#L38